src/GraphQL/Types/Scalars/TimeSpanMillisecondsGraphType.cs

```diff
@@ -18,21 +19,14 @@ public TimeSpanMillisecondsGraphType()
                 "The `Milliseconds` scalar type represents a period of time represented as the total number of milliseconds.";
```

Comment:
```suggestion
                "The `Milliseconds` scalar type represents a period of time represented as the total number of milliseconds in range [-922337203685477, 922337203685477].";
```

What do you think if we prohibit negative values for `Milliseconds` and `Seconds`? Now both accept negative values.

> represents a period of time

What is period of time?

`TimeSpan.FromMilliseconds(-100)` works fine and yields `{-00:00:00.1000000}`. Do we need to continue to take negative values?

Reply:
I don’t see a reason to prohibit negative numbers. We don’t know how the field will be used. Perhaps it’s a time offset from DOMContentLoaded, and a negative value is used to log the negative time offset indicating when the JavaScript started executing. Could be anything. (Which is why also I would suggest we change the type to double instead of long)

Reply:
(All JavaScript numbers are doubles anyway )

Reply:
OK. So I suggest to add only range in description.